code-challenges.js

```diff
@@ -8,62 +8,162 @@
 
 // Pseudo code:
 
-// Set one:
-const fruit1 = "apple"
-const fruit2 = "banana"
-// Expected outcome: "banana"
-
-// Set two:
-const fruit3 = "cherry"
-const fruit4 = "kiwi"
-// Expected outcome: "cherry"
-
-// --------------------1) Create a function that takes in a number and determines if the given number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below.
-
-// Pseudo code:
+// // Set one:
+// const fruit1 = "apple"
+// const fruit2 = "banana"
+// // Expected outcome: "banana"
+
+// // Set two:
+// const fruit3 = "cherry"
+// const fruit4 = "kiwi"
+// // Expected outcome: "cherry"
+
+// // --------------------1) Create a function that takes in a number and determines if the given number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below.
+
+// // Pseudo code:
+// // comment out the const temperatures and creat 3 different variables for temperature 1, 2, 3 and set them according to their respective temperatures
+// // create a function called boilingPoint
+// // create an argument and call it temp
+// // this is conditional statement, write else if with returns for each condition. If the temp is less than < 212, the returning output will be below boiling point.  If the temp is strictly at 212 the returning output is at boiling point, and else if any number above 212 output is returned above boiling point.
+// // // console log the boilingPoint and use the variables to see if numbers are below, at, or above boilingpoint
+var temperature1 = 42
+var temperature2 = 350
+var temperature3 = 212
+
+const boilingPoint = (temp) => {
+    if (temp < 212) {
+      return "Below boiling point";
+    } else if (temp === 212) {
+      return "At boiling point";
+    } else {
+      return "Above boiling point";
+    }
+  }
```

Comment:
No need to make new variables with the `var` declaration.
In your pseudo code, you said you were creating an argument temp, but it's actually a parameter that you are creating as `temp`.
Pay attention to the expected output: it is expecting the number that was passed in to be part of the string that gets returned!